src/www/ui/api/Controllers/GroupController.php

```diff
@@ -92,4 +92,52 @@ public function createGroup($request, $response, $args)
     }
     return $response->withJson($returnVal->getArray(), $returnVal->getCode());
   }
+
+  /**
+   * Change user permission
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function changeUserPermission($request, $response, $args)
+  {
+
+    // Extract all  prerequisites (dbManager , user_pk , new_permission , gropu_pk ) for this functionality
+    $dbManager = $this->dbHelper->getDbManager();
+    $user_pk = intval($request->getHeaderLine('userId'));
+    $newperm = intval($request->getHeaderLine('newPerm'));
+    $group_pk = intval($args['id']);
+
+    // Validate arguments
+    if (!$this->dbHelper->doesIdExist("users", "user_pk", $user_pk)) {
+      $info = new Info(404, "UserId doesn't exist", InfoType::ERROR);
+    } else if (!$this->dbHelper->doesIdExist("groups", "group_pk", $group_pk)) {
+      $info = new Info(404, "GroupId doesn't exist", InfoType::ERROR);
+    } else if ($newperm < -1) {
+      $info = new Info(400, "ERROR - permission can not be less than -1", InfoType::ERROR);
+    }else if($newperm > 2){
+      $info = new Info(400, "ERROR - permission can not be greater than 2", InfoType::ERROR);
+    }else {
```

Comment:
```suggestion
    } else if($newperm > 2) {
      $info = new Info(400, "ERROR - permission can not be greater than 2", InfoType::ERROR);
    } else {
```

to fix error: https://github.com/fossology/fossology/runs/7321278763?check_suite_focus=true#step:4:204